docs/gl_objects/topics.rst

```diff
@@ -46,3 +46,7 @@ Delete a topic::
 
     # or
     gl.topics.delete(topic_id)
+
+Merge a source topic into a target topic::
```

Comment:
Should it mention that it requires admin permission?

Reply:
Most of these endpoints here need admin access so we have this above, did you mean in addition to that?

> This endpoint requires admin access for creating, updating and deleting objects.

Reply:
Ah I didn't scroll back to see line 18.